src/Tizen.Applications.PackageManager/Tizen.Applications/Package.cs

```diff
@@ -364,6 +372,11 @@ internal static Package CreatePackage(IntPtr handle, string pkgId)
             {
                 Log.Warn(LogTag, "Failed to get whether package " + pkgId + " is preloaded or not");
             }
```

Comment:
The return value should be set to the err.
err = Interop.Package.PackageInfoIsUpdatePackage(handle, out package._isUpdated);